omnibus/config/patches/chef-dk/cli.patch

```diff
@@ -0,0 +1,17 @@
+diff --git a/lib/chef-dk/cli.rb b/lib/chef-dk/cli.rb
+index d789b075..adfc498e 100644
+--- a/lib/chef-dk/cli.rb
++++ b/lib/chef-dk/cli.rb
+@@ -95,9 +95,10 @@ BANNER
+     end
+ 
+     def show_version
+-      msg("Chef Development Kit Version: #{ChefDK::VERSION}")
++      msg("Chef Workstation Version: CHEF_WS_VERSION")
+ 
+-      ["chef-client", "delivery", "berks", "kitchen", "inspec"].each do |component|
++      msg("chef-dk version: #{ChefDK::VERSION}")
```

Comment:
I don't think we should be showing ChefDK version, since this is a replacement for ChefDK.

Reply:
👍

Reply:
I can buy that. I guess my feeling was we are not yet replacing the chef cli experience which is why I didn't clean up the output either. Let me fix it up with no DK and I might also indent the components to show more visually they are part of ws.